src/ansys/fluent/core/session.py

```diff
@@ -107,8 +107,8 @@ def __init__(
             fluent_connection (:ref:`ref_fluent_connection`): Encapsulates a Fluent connection.
             remote_file_handler: Supports file upload and download.
         """
-        self._remote_file_handler = remote_file_handler
         BaseSession.build_from_fluent_connection(self, fluent_connection)
+        self._remote_file_handler = remote_file_handler
```

Comment:
Can we set this inside `build_from_fluent_connection`?

Reply:
Actually, prior to current change it was here but then we moved it to `__init__`. Yes we can set it here.

Reply:
Done. Thank you.